provisioning_service_client/samples/enrollment_group_sample.py

```diff
@@ -28,7 +28,7 @@ def main():
     six.print_(eg)
 
     #make a Provisioning Service query
-    qs = QuerySpecification("*")
+    qs = QuerySpecification(query="*")
```

Comment:
>query [](start = 28, length = 5)

isn't switching to keyword args a breaking change?

Reply:
Switching to **enforced** keyword args is a breaking change. However, changing the syntax of a sample to favor keyword args (for consistency with the **new** objects which **do** use enforced keyword args) is not breaking since it supports both.